getting-started/kubernetes/hardway/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Up for the challenge? Calico the hard way takes you under the covers of a Calico installation.
+description: Go under the covers of an end-to-end Calico installation.
```

Comment:
I quite liked the quirkiness of the previous text. I would tweak it to say "an end-to-end" though as you've suggested.

Reply:
Fixed.